src/main/java/Victor/Storage/Storage.java

```diff
@@ -0,0 +1,76 @@
+package Victor.Storage;
+
+import Victor.TaskType.Deadline;
+import Victor.TaskType.Event;
+import Victor.TaskType.Task;
+import Victor.TaskType.Todo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+
+    File dataFile;
+    String filePath;
+    public Storage(String filePath) {
+        this.dataFile = new File(filePath);
+        this.filePath = filePath;
+    }
+
+    public ArrayList<Task> load() {
+        ArrayList<Task> currentList = new ArrayList<>();
+        try {
+            Scanner s = new Scanner(dataFile);
+            while (s.hasNext()) {
+                String nextLine = s.nextLine();
+                String[] inputs = nextLine.split("\\|");
+                String fileDataType = inputs[0].trim();
+                switch (fileDataType) {
+                    case "T" -> {
```

Comment:
Should there be indentation between switch and case?

Reply:
Fixed.